docs/detections/alert-suppression.asciidoc

```diff
@@ -81,3 +92,10 @@ With alert suppression, detection alerts aren't created for the grouped source e
 image::images/timeline-button.png[Investigate in timeline button, 200]
 
 * Alert details flyout — Select *Take action* -> *Investigate in timeline*.
+
+=== Alert suppression limit by rule type
+
+Except for custom query rules (which don't have an alert suppression limit), each rule type has a maximum number of alerts that can be suppressed:
```

Comment:
Maybe reorder; the focus point (each rule type has a maximum) feels a little buried behind two different qualifying phrases.
```suggestion
Some rule types have a maximum number of alerts that can be suppressed (custom query rules don't have a suppression limit):
```